css/components/Slider/slider.css

```diff
@@ -0,0 +1,63 @@
+.slider {
+  -webkit-appearance: none;
+  appearance: none;
+  background: transparent;
+  cursor: pointer;
+  width: 22rem;
+}
+
+.slider:focus {
+  outline: none;
+}
+
+/** Chromium based **/
+.slider::-webkit-slider-runnable-track {
+  background: var(--COLOR-GLOSSY-BLUE-100);
+  border-radius: 0.5rem;
+  height: 0.5rem;
```

Comment:
Can use px units instead of rem for less than 1rem

Reply:
Thanks for pointing this out, Shekhar. actually, I am using rems everywhere that's why I am maintaining that consistency. And Vikram also said we can use px for less than 10px values, but it is not necessary. that's why I am using rems.